src/brokers/add-broker/components/AddBrokerForm/FormView.component.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, useState } from 'react';
+import {
+  Form,
+  FormGroup,
+  TextInput,
+  Alert,
+  Button,
+  AlertGroup,
+  Grid,
+  GridItem,
+  ActionListItem,
+  ActionList,
+  // AlertActionCloseButton
+} from '@patternfly/react-core';
+import { useTranslation } from '../../../../i18n';
+import { Link } from 'react-router-dom';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+type FormViewProps = {
+  data: K8sResourceCommon;
+  handleNameChange: (fieldName: string, value: string) => void;
+};
+
+export const FormView: FC<FormViewProps> = ({ data, handleNameChange }) => {
+  const { t } = useTranslation();
+  const [showAlert, setShowAlert] = useState<boolean>(false);
+
+  const onSubmitHandler = (e: React.FormEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    if (data.metadata.name) {
+      setShowAlert(true);
+    }
+  };
+
+  return (
+    <Grid className="pf-u-mx-md">
+      <GridItem span={6}>
+        {showAlert && <Alert variant="info" isInline title={t('info_alert')} />}
```

Comment:
```suggestion
       <Alert variant="info" isInline title={t('info_alert')} />
```
This alert should display by default.We don't need to add any check.

Reply:
okay